examples/quickstart_tensorflow/README.md

```diff
@@ -0,0 +1,66 @@
+# Flower Example using Keras
+
+Flower is easy to use by only starting the Flower server and the Flower clients.
+
+It is recommended to use [pyenv](https://github.com/pyenv/pyenv)/[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and [poetry](https://python-poetry.org/docs/) to ensure the right version of libraries.
+
+After installing both tools you can start to set up everything.  
+
+## Setup your virtual environment
+
+First, you should setup your virtualenv and  install [Python Version 3.6](https://docs.python.org/3.6/) or above:
+
+```shell
+pyenv install 3.7.9
+```
+
+Create a virtualenv with:
+
+```shell
+pyenv virtualenv 3.7.9 keras-federated-3.7.9
+```
+
+Activate the virtualenv by creating a `.python-version` with the content:
+
+```shell
+keras-federated-3.7.9
+```
```

Comment:
```suggestion
```
I would skip this section and remove it as poetry by default creates a virtualenv. I just tried it out with the example and it worked well.

Reply:
This will be implemented in the next PR.